src/main/java/com/brandon3055/draconicevolution/client/gui/GUITeleporter.java

```diff
@@ -29,12 +29,14 @@
 import com.brandon3055.draconicevolution.common.lib.References;
 import com.brandon3055.draconicevolution.common.network.TeleporterPacket;
 
+import cpw.mods.fml.common.Loader;
 import cpw.mods.fml.relauncher.Side;
 import cpw.mods.fml.relauncher.SideOnly;
 
 @SideOnly(Side.CLIENT)
 public class GUITeleporter extends GuiScreen {
 
+    private static final int pearlsWithShift = Loader.isModLoaded("dreamcraft") ? 64 : 16;
```

Comment:
Variables, which are constants should be UPPER_CASE for convenience.

Reply:
Yeah, sorry, fixed